File: LLM-Translations/GPT-4/codenet/Go/Java/s633919449.java

import java.util.*;
import java.io.*;

public class s633919449 {
    static final int MOD = 1000000007;
    static BufferedReader br;
    static StringTokenizer st;

    public static void main(String[] args) throws IOException {
        br = new BufferedReader(new InputStreamReader(System.in));
        int N = getInt();

        switch (N % 10) {
            case 2:
            case 4:
            case 5:
            case 7:
            case 9:
                System.out.println("hon");
                break;
            case 0:
            case 1:
            case 6:
            case 8:
                System.out.println("pon");
                break;
            case 3:
                System.out.println("bon");
                break;
        }
    }

    static int getInt() throws IOException {
        return Integer.parseInt(next());
    }

    static String next() throws IOException {
        while (st == null || !st.hasMoreTokens()) {
            st = new StringTokenizer(br.readLine().trim());
        }
        return st.nextToken();
    }
}

File: LLM-Translations/GPT-4/codenet/Go/C/s633919449.c
#include <stdio.h>
#include <stdbool.h>
#include <string.h>
#include <math.h>

void main() {
    int N;
    scanf("%d", &N);

    switch (N % 10) {
    case 2:
    case 4:
    case 5:
    case 7:
    case 9:
        printf("hon\n");
        break;
    case 0:
    case 1:
    case 6:
    case 8:
        printf("pon\n");
        break;
    case 3:
        printf("bon\n");
        break;
    }
}

// 
